### PR TITLE
Change: Show all devices (not just root) in Config rows

### PR DESCRIPTION
**Pull request by Jskobos**

### CODE REVIEW

**Review by tiffwong — CHANGES_REQUESTED** [review]
If there are no configs, the VM Mode column wraps and the last column in the table headers looks funky
Screen Shot 2020-10-26 at 4 36 18 PM

File: packages/manager/src/features/linodes/LinodesDetail/LinodeAdvanced/LinodeConfigs_CMR.tsx [hunk 1]
```diff
@@ -99,13 +100,13 @@ const styles = (theme: Theme) =>
...
     },
     kernelColumn: {
-      width: '10%'
+      width: '20%'
     },
     interfacesColumn: {
-      width: '40%'
+      width: '20%'
     },
     rootDeviceColumn: {
```
Comment (DevDW) [inline packages/manager/src/features/linodes/LinodesDetail/LinodeAdvanced/LinodeConfigs_CMR.tsx:111]: Should probably rename this now that the column is no longer just for the root device

File: packages/manager/src/components/TableCell/TableCell_CMR.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     color: theme.cmrTextColors.tableStatic,
     fontSize: '.875rem',
     lineHeight: '1rem',
-    padding: '0px 15px',
+    padding: '10px 15px',
```
Comment (DevDW) [inline packages/manager/src/components/TableCell/TableCell_CMR.tsx:13]: This throws off some padding elsewhere in the app, e.g.

Screen Shot 2020-10-28 at 10 37 56 AM

**Review by DevDW — COMMENTED** [review]
The devices appear to be displaying accurately -- left a couple of comments

**Review by WilkinsKa1 — APPROVED** [review]
Changes look good; I did notice something slightly off with the table padding (sorry 😬 ) it's pretty edge-case (non-blocking) and only really noticeable when there are a certain number of disks:
Screen Shot 2020-10-28 at 1 04 59 PM

**Review by johnwcallahan — APPROVED** [review]

**Pull request merged**